CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# [2.145.0](https://github.com/revanced/revanced-patches/compare/v2.144.0...v2.145.0) (2022-12-18)
```

Comment:
This file is autogenerate, with the rest of the files except for the patch related files. Do not commit them.

Reply:
That's Y i closed the PR & create a new